backend/app/engine/octant_rewards/leftover/default.py

```diff
@@ -0,0 +1,10 @@
+from app.engine.octant_rewards.leftover import Leftover, LeftoverPayload
+
+
+class DefaultLeftover(Leftover):
+    def calculate_leftover(self, payload: LeftoverPayload):
```

Comment:
annotations for the returned values in the successors?

Reply:
ok